xbmc/dialogs/GUIDialogMediaFilter.cpp

```diff
@@ -393,13 +393,11 @@ void CGUIDialogMediaFilter::InitializeSettings()
     return;
   }
 
-  for (unsigned int index = 0; index < NUM_FILTERS; index++)
+  for (Filter filter : filterList)
```

Comment:
NUM_FILTERS macro now unused.

Reply:
this is definitely slower as it was before. it creates a copy that is not used if mediaType does not match.

there might be more of those little things we missed on review

Reply:
You are right, I thought I can smart and avoid to iterate over a reference as the variable is copied a few lines below anyway.